recipes/qpdf/all/conandata.yml

```diff
@@ -0,0 +1,8 @@
+sources:
+  "11.1.1":
+    url: "https://github.com/qpdf/qpdf/archive/refs/tags/v11.1.1.tar.gz"
+    sha256: "785edab622a1bc7e25e1537ad2c325005d48c5c7957f7abedff405deb80fa59a"
+patches:
+  "11.1.1":
+    - patch_file: "patches/0001-libqpdf-cmake-deps-jpeg-zlib.patch"
+    - patch_file: "patches/0002-exclude-unnecessary-cmake-subprojects.patch"
```

Comment:
Please, add `patch_description` and `patch_type: "conan"` to make easier to understand those patches. Docs: https://github.com/conan-io/conan-center-index/blob/master/docs/conandata_yml_format.md#patches-fields